src/consumer/frame.js

```diff
@@ -141,6 +149,26 @@ class Frame extends EventEmitter {
     if (this.wrapper.parentNode === this.container) {
       this.container.removeChild(this.wrapper);
       this.emit('xfc.unmounted');
+      this.cleanup();
+    }
+  }
+
+  /**
+   * Cleans up references of detached nodes by setting them to null
+   * to avoid potential memory leak
+   */
+  cleanup() {
+    this.iframe = null;
```

Comment:
i'd probably just move this inline into the one place its used.

Reply:
yeah, that's what I did initially, but found it hard to mock setting to nulls in tests (without mocking tests will fail). And that's the reason why I created a method.

Reply:
can we at least prepend the method name with __ or move this method off the class so that its "private".

Reply:
I was kinda hesitated on this because current eslint rule ([no-underscore-dangle](https://eslint.org/docs/2.0.0/rules/no-underscore-dangle)) discourages us to use prepended underscores. Their argument is that there's no truly private members in javascript. I'm on the fence. It looks like a matter of team preference and won't affect code performance, readability, or complexity. Personally I'd like to have underscores to indicate privacy as other language does (e.g. python), however, eslint rules can be considered as somewhat official style guide of JS maybe. Any thoughts you'd like to share?